server/views/components/actions-card/template.njk

```diff
@@ -0,0 +1,10 @@
+<div class="govuk-grid-column-one-third">
```

Comment:
I wonder if this component should not include the one-third div wrapper? 🤔 

My thinking is we might want several things in the right hand column (like we do at the moment) and/or we might want to use this component in a place where it is not a one-third column?

Thinking about whether we want other things in that column, we do currently, and this is what we do at the moment (`overviewTabContents.njk`):
```
<div class="govuk-grid-row">
  <div class="govuk-grid-column-two-thirds app-u-print-full-width">
    {% if isPostInduction %}
      {% include './overviewTabPostInduction.njk' %}
    {% else %}
      {% include './overviewTabPreInduction.njk' %}
    {% endif %}
  </div>

  <div class="govuk-grid-column-one-third app-u-print-full-width">
    {% include './functionalSkillsSidebar.njk' %}
    {% include './qualificationsAndAchievementsSidebar.njk' %}
  </div>

</div>
```
It feels right to me that the page defines the layout, and we just place components in in the relevant places?

Reply:
Done